inst/scripts/insert_image.R

```diff
@@ -0,0 +1,83 @@
+local({
+  txt_input = function(..., width = '100%') shiny::textInput(..., width = width)
+
+  blogfile = rstudioapi::getSourceEditorContext()
+  if (blogfile$path == '') stop('Please select the blog post source file before using this addin', call. = FALSE)
+
+  blogpath = normalizePath(blogfile$path)
+  imgdir = file.path(
+    gsub('/content/.*', '', blogpath),
+    'static',
+    dirname(gsub('.*content/', '', blogpath)),
+    paste0(xfun::sans_ext(basename(blogpath)), '_files')
+  )
+
+  shiny::runGadget(
+    miniUI::miniPage(miniUI::miniContentPanel(
+      shiny::fillRow(
+        shiny::fileInput('newimg', 'Image', placeholder = 'Select external image'),
```

Comment:
Just a random thought as I'm reviewing this PR:

Do you have an idea of how Cmd/Ctrl + V works on certain websites such as imgur.com? You can screenshot to the clipboard and "paste" on the page to upload. That has always been magic to me and I haven't done any research on it. It is probably not possible in the RStudio IDE (due to its relatively limited and old web browser kernel), though. If it can be made possible, people will definitely be thrilled.

Reply:
Err, I have no idea :P Sorry! I actually don't use imgur.com. I only tried it out a bit after your talk at JHU Biostats a few years ago (maybe 4? 5?)

Reply:
this can be done in R by call system to exec some script, in macos, script like this:
```
osascript -e "
  set theFile to (open for access POSIX file 'test.png' with write permission)
  try
    write (the clipboard as «class PNGf») to theFile
  end try
  close access theFile"
```

i have implemented a addins (https://github.com/xiaoa6435/RmdImgPaste) based on Timag's work